pkg/preflight/preflight.go

```diff
@@ -108,20 +108,22 @@ func Run(appID string, appSlug string, sequence int64, isAirgap bool, archiveDir
 			}
 			logger.Debug("preflight checks completed")
 
-			err = maybeDeployFirstVersion(appID, sequence, uploadPreflightResults)
+			isDeployed, err := maybeDeployFirstVersion(appID, sequence, uploadPreflightResults)
 			if err != nil {
 				err = errors.Wrap(err, "failed to deploy first version")
 				logger.Error(err)
 				return
 			}
 
-			if err := reporting.SendPreflightInfo(appID, int(sequence), false, false); err != nil {
-				logger.Debugf("failed to send preflights data to replicated app: %v", err)
-				return
+			if isDeployed {
```

Comment:
Why is preflight info not reported if version is not deployed?  Deploying is not required for preflight checks to have a status.

Reply:
the main goal is to report the status of the application when the user skips/waits for preflights when deploying

Reply:
the naming could be better i agree